### DISCUSSION TITLE
Internal Resolution ≠ Window Resolution (MUGEN Port)

### DISCUSSION THREAD

**Discussion by Ksylatarsh** [ikemen-engine/Ikemen-GO#1185] [Feature Request] [answered]
### Is your feature request related to a problem? Please describe.

I mostly have problems just sticking with one resolution for both because I want my games to use arcade resolution to prevent stuff like mixels and phantom pixels, in MUGEN this is possible by having Width/Height and GameWidth/GameHeight separated.

### Describe the solution you'd like

Port MUGEN's Width and Height system (GameWidth/GameHeight).

### Additional context

Probably keep screenshots too on internal resolution if possible?

### Search terms

width height gamewidth gameheight resolution

**Comment by K4thos** [comment, marked as answer]
added in #2185
